contracts/strategies/dnsVectorStrategy/DnsVectorStrategyStorageLib.sol

```diff
@@ -20,18 +23,23 @@ struct DnsVectorStorage {
     ERC20Upgradeable aaveBorrowToken;
     IERC20Upgradeable vAaveBorrowToken;
     IERC20Upgradeable ammPairDepositToken;
-    IERC20Upgradeable joeToken;
+    IERC20Upgradeable joeToken; // Obsolete.
     IPool aavePool;
     AaveProtocolDataProvider aaveProtocolDataProvider;
-    IJoeRouter02 traderJoeRouter;
-    IJoePair traderJoePair;
-    IVectorPoolHelperJoe vectorPoolHelperJoe;
+    IJoeRouter02 traderJoeRouter; // Obsolete.
+    IJoePair traderJoePair; // Obsolete.
+    IVectorPoolHelperJoe vectorPoolHelperJoe; // Obsolete.
     // The following variables are stored both in the storage and the base class,
```

Comment:
this comment originally applied only to the last 3 variables, can we somehow update this comment?